tests/explicit-nulls/pos/notnull/S.scala

```diff
@@ -0,0 +1,15 @@
+// Test the NotNull annotations are working in class files'
```

Comment:
This test exercises the java parser, not the class file, right?
If so, let's change the comment.

Reply:
Done